solution1.py

```diff
@@ -0,0 +1,48 @@
+import pandas
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+
+def ddSim(y,t0,r,K):
+    N=y[0]
+    dNdt=r*(1-N/K)*N
+    return [dNdt]
+    
+N0 = [10]
+K = 100
+listr = (-0.1,0.1, 0.4, 0.8, 1.0)
+times = range(0,600)
+modelOutput=pandas.DataFrame({"time":times,"r1":0, "r2":0, "r3":0, "r4":0, "r5":0})
+
+for i in range(len(listr)):
+    params = (listr[i], K)
+    modelSim=spint.odeint(func=ddSim,y0=N0,t=times,args=params)
+    modelOutput.iloc[:,i]=modelSim[:,0]
+    
+g1=ggplot(modelOutput,aes(x="time",y="r1"))+geom_line()+geom_line(aes(y="R2"))+geom_line(aes(y="R3"))+geom_line(aes(y="R4"))+geom_line(aes(y="R5"))
+g1+theme_classic()
+
```

Comment:
This code looks good except R2 should be r1. 

Also, for plotting, you can add color for example: 
ggplot(modelOutput,aes(x="time",y="r1",color="1"))+geom_line()+geom_line(aes(x="time",y="r2",color="50"))